docs/src/app/components/pages/components/DatePicker/ExampleInternational.jsx

```diff
@@ -4,26 +4,43 @@ import areIntlLocalesSupported from 'intl-locales-supported';
 
 let DateTimeFormat;
 
-// Use the native Intl if available
+/**
+ * Use the native Intl.DateTimeFormat if available, or a polyfill if not.
+ */
 if (areIntlLocalesSupported('fr')) {
```

Comment:
How did I miss that?! Although when I tested in Safari, which is the only mainstream desktop browser that doesn't support `Intl`, it worked fine (presumably since it's a binary condition).

Anyway, good catch (as always!). Updating now.